kernel/packages/shared/comms/v2/LighthouseWorldInstanceConnection.ts

```diff
@@ -95,6 +95,16 @@ export class LighthouseWorldInstanceConnection implements WorldInstanceConnectio
     return this.cleanUpPeer()
   }
 
+  analyticsData() {
+    // For now, these are private. Next version of peer library will make them public
+    return {
+      // @ts-ignore
+      connectedPeers: this.peer.fullyConnectedPeerIds().map(it => it.slice(-6)),
```

Comment:
Why do we send the last 6 characters of the id? Should we document this?

Reply:
This arised after speaking with @dnul. Since some of these lists could be very big, we should maybe use a part of the id. 

In general, we should do this all the way (in comms), but since that is more likely to cause some issues, I didn't want to go slicing ids in peer

Reply:
Alright! 👍

Reply:
Can we add a comment there? to avoid any confusion in the future

Reply:
Done!